backend/src/test/java/com/example/backend/controller/CarHealthControllerTest.java

```diff
@@ -55,8 +55,8 @@ class CarHealthControllerTest {
     void getAllCars_shouldGetAllCars() throws Exception {
```

Comment:
Hey Marija. i really like the fact that you put so much effort into your tests. Your controller test uses the @SpringBootTest annotation, which is perfectly correct as you want to do an integration test here. I would try to build an integration test without mocking, i.e. implement the actual dependencies with @Autowired and call them directly. Since we are already using flapdoodle, using the database should not be a problem.


Translated with DeepL.com (free version)

Reply:
Thank you for the comment! I'll try to implement it to the tests in the next branch